README.md

```diff
@@ -103,4 +103,11 @@ For the git submodules, you will have to undo the removal of the `yarn.lock` fil
 
 Make PRs for the monorepo and the submodules.
 
-Make a Github release
+Tag the commit with the appropriate version, and then create a github release with a changelog against the previous version https://github.com/hyperlane-xyz/hyperlane-monorepo/releases/new
+
+### Release Agents
+
+- Tag the commit with the appropriate version
```

Comment:
I think we should switch to dates instead of versions. Since we are packaging 3 binaries into one release, defining "breaking changes" by semver is a bit clunky. For example in the 1.0.1 release, the scraper deserves a `MAJOR` version bump as it changed the database format and requires re-indexing. The validator deserves a `PATCH` as it only had minor improvements. The relayer deserves a `MINOR` as new features were added such as `defaultSigner` and sepolia but it should not change how it is is run.